### PR TITLE
Allow multiple KeysMap files to be specified

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

**Comment by slawekjaranowski** [comment]
fix for #246

File: src/site/markdown/keysmap-multiple.md.vm [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Multiple KeysMpa configuration
```
Comment (pavelhoral) [inline src/site/markdown/keysmap-multiple.md.vm:1]: Typo: KeysMap.

File: src/site/markdown/keysmap-multiple.md.vm [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+==============================
+
+You can use a multiple source of `keysMap` file with the plugin.
```
Comment (pavelhoral) [inline src/site/markdown/keysmap-multiple.md.vm:4]: Grammar: _You can use multiple sources_

File: src/site/markdown/keysmap-multiple.md.vm [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Simply way
```
Comment (pavelhoral) [inline src/site/markdown/keysmap-multiple.md.vm:6]: Grammar: _Basic example_ or _Basic usage_ (i don't think _simply way_ is quite correct)

File: src/site/markdown/keysmap-multiple.md.vm [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                <exclude>
+                    <pattern>org\.example2\..*</pattern>
+                </exclude>
+            </keysMapLocation>
+        </keysMapLocations>
+    </configuration>
+</plugin>
+```
+
+Yuo can define many `include` and `exclude` tags for specific `keysMpa`.
```
Comment (pavelhoral) [inline src/site/markdown/keysmap-multiple.md.vm:60]: Typo: _You_ + _keysMap_

File: src/site/markdown/keysmap-multiple.md.vm [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+| item | description | default |
+| ------ | ----- | ----- |
+| include,exclude / pattern | it is a java Pattern which will be applied to artifact in keysMap | .* |
+| include,exclude / value   | it can be a special value like, any,badSig,noKey,noSig | any |
```
Comment (pavelhoral) [inline src/site/markdown/keysmap-multiple.md.vm:65]: Comma and spaces? Not sure what was the intended format.

> like _any_, _badSig_, _noKey_, _noSig_

File: src/site/site.xml [hunk 6]
```diff
@@ -23,6 +23,7 @@
             <item name="Introduction" href="index.html"/>
             <item name="Goals" href="plugin-info.html"/>
             <item name="KeysMap file format" href="keysmap-format.html"/>
+            <item name="Multiple KeysMpa configuration" href="keysmap-multiple.html"/>
```
Comment (pavelhoral) [inline src/site/site.xml:26]: Typo: _KeysMap_

**Review by pavelhoral — COMMENTED** [review]
I thought I might give it a look and I like it. Found just a few typos (sorry for nitpicking :))

**Comment by slawekjaranowski** [comment]
@pavelhoral  thanks, I've fixed

**Pull request merged**